shiny/www/shared/bslib/_version.json

```diff
@@ -1,5 +1,5 @@
 {
   "note!": "Generated by scripts/htmlDependencies.R: do not edit by hand",
   "package": "bslib",
-  "version": "0.7.0.9000 (rstudio/bslib@c5244cfae75c434e5fae6091820c13b62de53fd7)"
+  "version": "0.7.0.9000 (rstudio/bslib@2b4bfc5a4d2e40b48338a38361ae8d453001edf8)"
```

Comment:
Small nit: could run again to bring up to latest ([fd5c97d](https://github.com/rstudio/bslib/commit/fd5c97de3b6957c6effb295f24e1739ef6f3709f))